Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1003,6 +1003,30 @@ void TextEditor::keydown_event(KeyEvent& event)
         return;
     }
 
+    if (event.ctrl() && event.key() == KeyCode::Key_Slash) {
+        if (m_highlighter != nullptr) {
+            String comment;
+            switch (m_highlighter->language()) {
+            case Syntax::Language::Cpp:
+            case Syntax::Language::JavaScript:
+                comment = "// ";
+                break;
+            case Syntax::Language::INI:
+                comment = "; ";
+                break;
+            case Syntax::Language::SQL:
+                comment = "-- ";
+                break;
+            default:
+                comment = "";
+            }
+            auto const old_cursor = m_cursor;
+            TextPosition start = { m_cursor.line(), current_line().first_non_whitespace_column() };
+            execute<InsertTextCommand>(comment, start);
+            set_cursor(old_cursor);
```

Comment:
You should `return;` here so that the event.ignore() call isn't made below, which will cause this event to continue bubbling up to parent widgets.